example/use_cases/employer_university_flow/employer_university_flow/util.go

```diff
@@ -0,0 +1,315 @@
+package employer_university_flow
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/TBD54566975/ssi-sdk/credential"
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/cryptosuite"
+	"github.com/TBD54566975/ssi-sdk/did"
+	"github.com/TBD54566975/ssi-sdk/util"
+	"github.com/sirupsen/logrus"
+)
+
+var cw = CustomStepWriter{}
+
+var (
+	UnsupportedDIDErorr = errors.New("unsupported Method for DID")
+)
+
+// Resolves a DID
+// Right the current implementation ssk-sdk does
+// not have a universal resolver.
+// https://github.com/decentralized-identity/universal-resolver
+// is a case where a universal resolver is implemented,
+// but the resolution would need to be hooked with the sdk.
+//  TODO (andor): Should exist a universal resolution method somewhere
+// in the actual SDK
+func resolveDID(didStr string) (*did.DIDDocument, error) {
```

Comment:
this would be a good addition to the `did` package

Reply:
ahhhh! OK. I will add a PR with this in the ssi-sdk!

Reply:
done.